Project.toml

```diff
@@ -50,17 +51,23 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [weakdeps]
 Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
+Convex = "f65535da-76fb-5f13-bab9-19810c17039a"
+ECOS = "e2685f51-7e38-5353-a97d-a921fd2c8199"
 
 [extensions]
 TrixiMakieExt = "Makie"
+TrixiConvexECOSExt = "Convex"
 
 [compat]
 CodeTracking = "1.0.5"
 ConstructionBase = "1.3"
+Convex = "0.15.4"
 DataStructures = "0.18.15"
+DelimitedFiles = "1"
 DiffEqBase = "6 - 6.143"
 DiffEqCallbacks = "2.25"
 Downloads = "1.6"
+ECOS = "1.1.2"
```

Comment:
Are the compat entries for `Convex` and `ECOS` still required after defining the extension?
(I genuinely don't know)

Reply:
Yes! According to this documentation: https://pkgdocs.julialang.org/v1/creating-packages/#Weak-dependencies